docs/development/configuration.md

```diff
@@ -159,11 +159,20 @@ steps:
   - type: installComposerDependencies
   - type: collectBuildInstructions
   - type: processSourceFiles
+    options:
+      fileConditions:
+        # You can define a Twig template to render...
+        - path: composer.json.twig
+          if: 'features["composer"]'
+        # ... or use static files
```

Comment:
To have some variance and also more complex examples, I'd suggest the following instead (using a static file with condition and renaming):
```
  # ... or use static files
  - path: example-v3.conf
    if: 'features["example"] && example["version"] == "3"'
    target: example.conf
```